osi/include/cloe/component/osi_sensor.hpp

```diff
@@ -41,6 +42,22 @@ class OsiSensor : public Component {
    * Return OSI-SensorData
    */
   [[nodiscard]] virtual const osi3::SensorData& get() const = 0;
+
+  /**
+   * Writes JSON representation into j.
+   */
+  Json active_state() const override {
+    return Json{
+        {"sensor_data", this->get_json_str()},
```

Comment:
This is going to be a string of JSON, which is not the same thing as JSON. Is that what you want?

Reply:
It sucks really hard, but we're probably going to have to deserialize the JSON only for it to be serialized again later.

Reply:
I re-implemented this in a fixup.